github/repos_rules.go

```diff
@@ -378,7 +378,7 @@ type RepositoryRulesetCodeScanningRule struct {
 
 // RuleCodeScanningParameters defines parameters for code scanning rules.
 type RuleCodeScanningParameters struct {
-	CodeScanningTools []CodeScanningTool `json:"code_scanning_tools"`
+	CodeScanningTools []CodeScanningTool `json:"code_scanning_tools,omitempty"`
```

Comment:
```suggestion
	CodeScanningTools []*CodeScanningTool `json:"code_scanning_tools,omitempty"`
```

Reply:
sorry missed this, correcting it right away